### PR TITLE
[CUBRIDQA-60] fix unstable for localinvoker and junit testing

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 
fix unstable for localinvoker and junit testing

### CODE REVIEW

File: CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java [hunk 1]
```diff
@@ -152,6 +163,20 @@ private static String getAllTestCaseScripts(String dir) {
...
 	
+	private boolean isValidTestMethod(String className){
+		boolean ret = false;
+		if(className == null || className.length() == 0) return ret;
+		
+		if (className.toUpperCase().indexOf("TEST") >= 0
+				&& className.indexOf("teSt") == -1
+				&& className.indexOf("tesT") == -1
+				&& className.indexOf("tSet") == -1
+				&& EXCLUDED_METHOD_LIST.indexOf(className) == -1) {
```
Comment (fanzaiqiang) [inline CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java]: as is:
EXCLUDED_METHOD_LIST.indexOf(className) == -1) {

to be:
EXCLUDED_METHOD_LIST.indexOf(","+className+",") == -1) {

**Comment by supler** [comment]
@fanzaiqiang 
add ignore annotation support

**Comment by fanzaiqiang** [comment]
@supler 

Please go ahead to merge it. Thanks.

**Comment by supler** [comment]
@fanzaiqiang 

print progress on console and save case runtime log into file

File: CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java [hunk 2]
```diff
@@ -45,47 +47,53 @@ public JdbcLocalTest(Context context){
...
-		log = new Log(CommonUtils.concatFile(context.getCurrentLogDir(), "test_jdbc.log"), false);
+		runCaseLog = new Log(CommonUtils.concatFile(context.getCurrentLogDir(), "run_case_details.log"), false);
 		this.context.setTestCaseRoot(com.navercorp.cubridqa.common.CommonUtils.translateVariable(this.context.getTestCaseRoot()));
 	}
 
 	private void start(){
 		String scenarioDir = this.context.getTestCaseRoot();
 		try {
 			if (scenarioDir == null || scenarioDir.length() <= 0) {
-				this.log.println("[ERROR] Scenario is not configured.");
```
Comment (fanzaiqiang) [inline CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java]: this.runCaseLog.println("[ERROR] Scenario is not configured.");

It's better to print the info to console.

File: CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java [hunk 3]
```diff
@@ -122,14 +135,26 @@ private void runTests(JdbcCaseMethodBean caseMethodBean){
...
 			}
-			context.getFeedback().onTestCaseStopEvent(caseClassPackageFullName + "=>" +  methodName, isSucc, runTime, res, "local", false, false, ingoreCount==0 ? Constants.SKIP_TYPE_NO : Constants.SKIP_TYPE_BY_INVALID_UNITCASE , 0);
+			this.runCaseLog.println(baos.toString());
+			this.runCaseLog.println("===================================================");
+			System.setOut(origStdout);
+			System.setErr(origStderr);
+			System.out.println(res);
+			System.out.println();
+			System.out.println("===================================================");
+			context.getFeedback().onTestCaseStopEvent(caseFile + ":" + methodName + "()", isSucc, runTime, res, "local", false, false, Constants.SKIP_TYPE_NO , 0);
```
Comment (fanzaiqiang) [inline CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java]: as is:
context.getFeedback().onTestCaseStopEvent(caseFile + ":" + methodName + "()", isSucc, runTime, res, "local", false, false, Constants.SKIP_TYPE_NO , 0);

to be:
if test is successful, make res to null.
We needn't record messages when a test case passed.

**Comment by supler** [comment]
@fanzaiqiang 

refine logs

**Comment by fanzaiqiang** [comment]
@supler 

I think this can be committed.
Then let's have a run_jdbc.sh script for daily regression test in new pull request.

**Pull request merged**